src/components/Disputes/DisputeInfo.js

```diff
@@ -256,18 +256,42 @@ function Field({ label, value }) {
       >
         {label}
       </h2>
-      {isAddress(value) ? (
-        <div
-          css={`
-            display: flex;
-            align-items: flex-start;
-          `}
-        >
-          <LocalIdentityBadge
-            connectedAccount={addressesEqual(value, wallet.account)}
-            entity={value}
-          />
-        </div>
+
+      {typeof value === 'string' ? (
+        value.split('\n').map((line, i) => (
+          <React.Fragment key={i}>
+            {transformAddresses(line, (part, isAddress, index) =>
+              isAddress ? (
+                <span title={part} key={index}>
+                  {' '}
```

Comment:
Should we remove this?

Reply:
oh yes those are getting added when i save